enhancements/oc/cli-manager.md

```diff
@@ -0,0 +1,233 @@
+---
+title: cli-manager
+authors:
+  - "@sallyom"
+  - "@deejross"
+reviewers:
+  - "@soltysh"
+  - "@jwmatthews"
+approvers:
+  - "@soltysh"
+  - "@sferich888"
+  - "@deads2k"
+  - "@spadgett"
+creation-date: 2021-10-06
+last-updated: 2021-10-18
+status: implementable
```

Comment:
nit: update with latest [template](https://github.com/openshift/enhancements/blob/master/guidelines/enhancement_template.md), I'm especially missing:
```
api-approvers:
  - # probably @deads2k here, pls ping him about it 
tracking-link:
  - https://issues.redhat.com/browse/WRKLDS-169
see-also:
  - "/enhancements/installer/component-selection.md"
```
since the latter is connected with this controller having the ability to be turned off, among others.

Reply:
Alternatively the initial version of API can be approved by me as well, and we'll update later on.